Mathlib/LinearAlgebra/VectorSpace.lean

```diff
@@ -0,0 +1,23 @@
+/-
+Copyright (c) 2024 Julian Berman. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Julian Berman
+-/
+import Mathlib.Algebra.Field.Defs
+import Mathlib.Algebra.Module.Pi
+import Mathlib.Tactic.Variable
+
+/-!
+# Vector spaces
+
+This file provides `VectorSpace` as a simple alias for a module over a field using the
+`variable_alias` attribute.
+-/
+
+/-- A vector space is a module over a field. -/
+@[variable_alias]
+structure VectorSpace (k V : Type*)
+  [Field k] [AddCommGroup V] [Module k V]
```

Comment:
The docstring should underline the fact that this is not a class and it's here to be used with a `variable?` command, not as a `[VectorSpace K V]` assumption in a theorem.

Reply:
(I agree with this) but it will apply to every use of `variable_alias`, correct? In which case it seems maybe to me like perhaps `variable_alias` should automatically append that sort of message to the docstring of the declaration you use it on, presuming that should be possible, no?